frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -99,3 +99,7 @@ export enum REQUESTER_FILTER {
   USER = 'user',
   SYSTEM = 'system',
 }
+
+export const CLUSTER_VERSION_DEFAULT_UPSTREAM_SERVER_URL =
+  'https://api.openshift.com/api/upgrades_info/v1/graph';
+export const CLUSTER_VERSION_CUSTOM_UPSTREAM_SERVER_URL_PLACEHOLDER = 'https://';
```

Comment:
This seems like an odd placeholder. Generally you'd want the placeholder to be a valid value, so I'd use something with example.com like

```suggestion
export const CLUSTER_VERSION_CUSTOM_UPSTREAM_SERVER_URL_PLACEHOLDER = 'https://example.com/api/upgrades_info/v1/graph';
```